.github/workflows/release.yml

```diff
@@ -111,22 +113,23 @@ jobs:
           jq -r .token)"
           echo "::add-mask::$TOKEN"
           echo "::set-output name=token::$TOKEN"
-      - uses: actions/download-artifact@v3
+      - name: Checkout
+        uses: actions/checkout@v3
         with:
-          name: rule-schema-yaml
-          path: rule_schema.yaml
+          submodules: true
+          token: ${{ steps.token.outputs.token }}
       - name: Upload Schema Files
         id: upload-semgrep-schema-files
         env:
           GITHUB_TOKEN: "${{ steps.token.outputs.token }}"
         run: |
-          gh release --repo returntocorp/semgrep-interfaces upload ${{ steps.get_version.outputs.VERSION }} rule_schema.yaml
+          gh release --repo returntocorp/semgrep-interfaces upload ${{ needs.create_release.outputs.VERSION }} cli/src/semgrep/semgrep_interfaces/rule_schema.yaml
```

Comment:
I actually have a change that rename this file to rule_schema_v1.yaml, so we might need to take care when we merge it.

Reply:
Check, good catch. Do we want to dynamically look for and upload `rule_schema*.yaml`?

Reply:
I saw your PR @aryx and will rebase this off develop once that goes in.

Reply:
Fixed. Also removed one other unused step.